assignment1/kthInLinkedList.py

```diff
@@ -0,0 +1,41 @@
+class Node:
+    def __init__(self, _key = 0, _next = None):
+        self.key = _key
+        self.next = _next
+
+    def __str__(self):
+        return str(self.key)
+
+def kthInLinkedList(linkedList, k):
+    if k < 0:
+        return None
+    ptr1 = ptr2 = linkedList
+    cnt = -1
+    while cnt != k:
```

Comment:
It is not immediately clear what this code is doing.
Maybe adding a couple of comments would help here.